### PR TITLE
Elixir 1.17 and OTP 27.0

### PR DESCRIPTION
**Pull request by krns**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -8,7 +8,7 @@ defmodule Backpex.MixProject do
     [
       app: :backpex,
       version: @version,
-      elixir: "~> 1.12",
+      elixir: "~> 1.15",
```
Comment (krns) [inline mix.exs:11]: @pehbehbeh @Flo0807 I think we should bump this version, but I'm not sure which version we want to "officially" support.

Comment (krns) [reply mix.exs]: Bumped it to 1.16

Comment (Flo0807) [reply mix.exs]: 👍

**Comment by Flo0807** [comment]
@krns can you update this PR with the new erlang / elixir version and fix the conflict?

**Pull request merged**